### PR TITLE
Add new target EASP-RRIOTF722

### PR DESCRIPTION
**Pull request by ws998116**

### CODE REVIEW

**Comment by nerdCopter** [comment]
is it possible to add the `timer` and `dma` comments similar to other existing targets?  (does not affect usage/merging, just an unimportant request).

File: configs/default/EASP-RRIOTF722.config [hunk 1]
```diff
@@ -0,0 +1,92 @@
+
+#define USE_GYRO
+#define USE_GYRO_SPI_MPU6000
+#define USE_ACC
+#define USE_ACC_SPI_MPU6000
+#define USE_BARO
+#define USE_BARO_SPI_BMP388
+#define USE_FLASH
+#define USE_FLASH_W25Q128FV
+
```
Comment (nerdCopter) [inline configs/default/EASP-RRIOTF722.config:10]:
```suggestion
```

**Comment by nerdCopter** [comment]
@haslinghuis , does line 1 need to be a Betaflight version?

File: configs/default/EASP-RRIOTF722.config [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
```
Comment (haslinghuis) [inline configs/default/EASP-RRIOTF722.config:1]:
```suggestion
# Betaflight / STM32F7X2 (S7X2) 4.4.3 Nov 14, 2023 / 22:53:32 (738127e) MSP API: 1.45
#mcu STM32F7X2
```

File: configs/default/EASP-RRIOTF722.config [hunk 3]
```diff
@@ -0,0 +1,92 @@
+# Betaflight / STM32F7X2 (S7X2) 4.4.3 Nov 14, 2023 / 22:53:32 (738127e) MSP API: 1.45
+#mcu STM32F7X2
```
Comment (nerdCopter) [inline configs/default/EASP-RRIOTF722.config:2]:
```suggestion
#mcu STM32F7X2

```
i wouldn't mind an empty-line here, but such is getting picky.

**Review by nerdCopter — DISMISSED** [review]

**Review by nerdCopter — APPROVED** [review]

**Review by haslinghuis — APPROVED** [review]
Before merging - we need https://github.com/betaflight/unified-targets/pull/1277

**Comment by ws998116** [comment]
Please leave this open for now, there will be some changes.

**Comment by ot0tot** [comment]
Is there a typo in the config? Gyro, flash, OSD, and baro should not all be on the same SPI bus.

**Comment by ws998116** [comment]
@ot0tot we are moving the baro to I2C, but the rest will share SPI.

**Comment by ot0tot** [comment]
> @ot0tot we are moving the baro to I2C, but the rest will share SPI.

Can you move flash and OSD to their own SPI bus? You cannot have all of these perepherials on the same bus.

**Comment by ws998116** [comment]
@ot0tot are you suggesting to have three separate SPI buses? One for each peripheral?

**Comment by ot0tot** [comment]
> @ot0tot are you suggesting to have three separate SPI buses? One for each peripheral?

Correct. Refer to other similar target configs for examples.

**Comment by haslinghuis** [comment]
Closing as we no longer accept new unified targets. Please create a config in https://github.com/betaflight/config repo with requested SPI assignments.

**Pull request closed without merging**